src/magic_folder/create.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2020 Least Authority TFA GmbH
+# See COPYING for details.
+
+"""
+Implements the magic-folder create command.
+"""
+import codecs
+import os
+
+from twisted.web.client import (
+    Agent,
+)
+
+from hyperlink import (
+    DecodedURL,
+)
+
+from .invite import (
+    magic_folder_invite as _invite
+)
+
+from .join import (
+    magic_folder_join as _join
+)
+
+from .common import (
+    get_node_url,
+    tahoe_mkdir,
+)
+
+from treq.client import (
+    HTTPClient,
+)
+
+from twisted.internet.defer import (
+    inlineCallbacks,
+    returnValue
+)
+from .frontends.magic_folder import (
+    load_magic_folders,
+    maybe_upgrade_magic_folders,
+)
+
+from allmydata.scripts.common import get_aliases
+from allmydata import uri
+from allmydata.util import fileutil
+from allmydata.util.encodingutil import quote_output
+
+# Until there is a web API, we replicate add_alias.
+# https://tahoe-lafs.org/trac/tahoe-lafs/ticket/3286
+def add_line_to_aliasfile(aliasfile, alias, cap):
+    # we use os.path.exists, rather than catching EnvironmentError, to avoid
+    # clobbering the valuable alias file in case of spurious or transient
+    # filesystem errors.
+    if os.path.exists(aliasfile):
+        with codecs.open(aliasfile, "r", "utf-8") as f:
+            aliases = f.read()
+            f.close()
```

Comment:
Don't need to explicitly `close` here; the context-manager will close when the "`with`" block ends (if you exit it "normally" or via exception etc).